dataprofiler/profilers/profile_builder.py

```diff
@@ -1352,9 +1349,7 @@ def tqdm(l):
         
         # If there are no minimum true samples, you can sort to save time
         if min_true_samples in [None, 0]:
-            # If there's a sample size, truncate
-            if sample_size is not None:
-                sample_ids[0] = sample_ids[0][:sample_size]
+            sample_ids[0] = sample_ids[0][:sample_size]
```

Comment:
does that mean when sample_size is None, we take all samples? Does that the meaning of sample_size?

Reply:
sample_size cannot be `None`, sample_size is always set within this function